### PR TITLE
Integration tests archive

### PR DESCRIPTION
**Pull request by kiran-chaudhary**
### Description of Change ###

Added `archive` integration tests.
### PR Checklist ###

Did you remember to do the below?

- [NA ] Add unit tests to verify this change
- [ NA] Add an entry to CHANGELOG.md describing this change
- [ Na] Add docstrings for any new public parameters / methods / classes

### CODE REVIEW

File: integration/test_archive.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+from py42.exceptions import Py42ForbiddenError
+
+device_guid = "935873453596901068"
+device_id = "110395"
+org_id = "2689"
+user_id = "176457"
+destination_device_guid = "673679195225718785"
+archive_guid = "912100293346985227"
+path = "C:/Users/QA/Downloads/spatel_add.csv"
```
Comment (unparalleled-js) [inline integration/test_archive.py:13]: Can we pick a file that does not contain a user's name in it?

Comment (unparalleled-js) [reply integration/test_archive.py:13]: `spatel` is what I am referring to

Comment (kiran-chaudhary) [reply integration/test_archive.py:13]: Shall we discuss this [here](https://github.com/code42/py42/pull/259#discussion_r547085417).

Comment (unparalleled-js) [reply integration/test_archive.py:13]: all you we need to do is pick another file! There are other files without his name in it

**Review by unparalleled-js — CHANGES_REQUESTED** [review]
Is there not another file we can choose that does not contain his name?

File: integration/test_archive.py (hunk 1 above)
Comment (kiran-chaudhary) [reply integration/test_archive.py]: changed.

**Review by unparalleled-js — APPROVED** [review]

**Pull request merged**